functions/parser.scm

```diff
@@ -239,10 +213,11 @@
 				 [node2-id (cog-name (cog-outgoing-atom listlink 1))]
 				 [node-id (if (not (node-exists? node1-id atoms)) node1-id (if (not (node-exists? node2-id atoms)) node2-id))]
 				 [node-id (if (unspecified? node-id) node1-id node-id)] ;TODO Weird issue where a node becomes unspecified for no reason
-;				 [node-name (get-node-info-from-biogrid main-annotation "name" node-id)]
+;				 [node-name (get-node-info-from-biogrid (cog-outgoing-set main-annotation)"name" node-id)]
 				 [node-name ""]
```

Comment:
`node-name` should not be empty string. It should be set to the value of the `node-id` if it isn't available. This is because when the JSON is exported in the visualizer and import the the Cytoscape Desktop app, it shows the nodes with empty boxes.